src/dojos/service.js

```diff
@@ -24,13 +25,20 @@ const DojosService = {
       },
     );
   },
-  getDojosByLatLong: (lat, lon) => Vue.http.post(`${Vue.config.apiServer}/api/2.0/dojos/search-bounding-box`, {
-    query: {
-      lat,
-      lon,
-      radius,
-    },
-  }),
+  async getDojosByLatLong(lat, lon) {
+    const dojos = { body: [] };
+    const response = await Vue.http.post(`${Vue.config.apiServer}/api/2.0/dojos/search-bounding-box`, {
+      query: {
+        lat,
+        lon,
+        radius,
+      },
+    });
+    response.body.forEach((snakeCaseDojo) => {
+      dojos.body.push(mapKeys(snakeCaseDojo, (value, key) => camelCase(key)));
```

Comment:
you're losing everything which is not "body".
I'd recommend to clone response.body, delete response.body, and then replacing response.body with the newly formatted.